client/src/components/categorical/value.js

```diff
@@ -173,6 +179,27 @@ class CategoryValue extends React.Component {
     });
   };
 
+  isAddCurrentSelectionDisabled(category, value) {
+    /*
+    disable "add current selection to label", if one of the following is true:
+    1. no cells are selected
+    2. all currently selected cells already have this label, on this category
+    */
+    const { crossfilter, world } = this.props;
+
+    // 1. no cells selected?
+    if (crossfilter.countSelected() == 0) {
```

Comment:
Prefer `===` https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons

Reply:
fully agree!